app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers ask for proof of identification"],
+    notes: [
+      "Ideally this field would be combined with given name but recorded separately to support legacy student record systems.",
+      "HESA also ask for ‘Immediately prior surname’ to “to allow matching of records where a student has changed surname” but we don’t capture this information.",
+      "Should we allow changes post-submission?"
+    ]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Date of birth",
+  items: {
+    question: "Date of birth",
+    required: "Yes",
+    api: {
+      param: "candidate.date_of_birth",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "use this when performing a DBS check"
+    }, {
+      who: ["hesa"],
+      what: "needs this to be in the format YYYY-MM-DD in order to link to other records (e.g. higher education, wider than just ITT)"
+    }, {
+      who: ["dfe"],
+      what: "uses this for record matching and statistical analysis, e.g. looking at application rates of over 35 year olds (something the minister has historically been interested in)"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers ask for proof of identification"],
+    notes: ["We need to collect year, but do we need to show it to providers?"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Nationality",
+  items: {
+    question: "Nationality",
+    required: "Yes",
+    api: {
+      param: "candidate.nationality[]",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "may use this to assess if a candidate is [eligibile for student finance](https://www.gov.uk/student-finance/who-qualifies)"
+    }, {
+      who: ["hesa"],
+      what: "requires this “to respond to Parliamentary Questions, Written Questions, FOI requests on complex citizenship/nationality issues. To be able to perform analysis on trainee demographics at the start of an academic year.”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers ask for proof of identification"],
+    notes: ["We need to collect year, but do we need to show it to providers?"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Second nationality",
+  items: {
+    question: "Add another nationality → Second nationality",
+    required: "No",
+    api: {
+      param: "candidate.nationality[]",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    notes: ["We needn’t ask for additional nationalities; dual nationals can enter their British or EEA-derived nationality"]
```

Comment:
There's also something here about multi nationals having to make a choice about which other nationality/nationalities they show. We don't help them with this. Adding all might simplify.

Reply:
Updated note.